aws/basic-eks-cluster/blueprint.yaml

```diff
@@ -123,6 +125,29 @@ spec:
     - p3.8xlarge
     - p3.16xlarge
 
+  - name: AddIAMUser
```

Comment:
why are we asking these questions? we are provisioning this anyway right?

Reply:
also these 3 variables are not used anywhere?

Reply:
nope. only if someone want to add a user he can, we do not provision  users anymore

Reply:
look at the code dude, this is not used anywhere

Reply:
https://github.com/xebialabs/blueprints/blob/qpi-1/aws/eks-provisioning/kubernetes/aws-auth-cm.yaml.tmpl something probably went wrong with the merge

Reply:
i'm gonna clean this shit up, we dont want too many questions here anyway @vpartington wants these basic ones to be simple

Reply:
up to you